jobs/flannel-job.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cyclonus
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - command:
+            - ./cyclonus
+            - generate
```

Comment:
Based on our slack convo, you could increase the time cyclonus waits for pods to come up by adding an `--pod-creation-timeout-seconds=120` arg here.